test/test_utils.sh

```diff
@@ -96,6 +97,19 @@ build_and_push_pre_upgrade_check_image() {
     travis_fold_end build_and_push_pre_upgrade_check_image
 }
 
+build_and_push_reporter_image() {
+    image_tag=$1
+    cache_image=$2
+    travis_fold_start build_and_push_reporter_image $image_tag
+
+    docker build -q -t $image_tag -f $ROOT/cmd/reporter/Dockerfile.reporter --cache-from ${cache_image} --build-arg GITCOMMIT=$(getGitCommit) --build-arg BUILDDATE=$(getDate) --build-arg BUILDVERSION=$(getVersion) .
+
+    gcloud_login
+
+    gcloud docker -- push $image_tag &
+    travis_fold_end build_and_push_reporter_image
```

Comment:
1. We are not using the GCR and Travis both
2. I think we should not doing analytics part as part of test - that will skew results.